### ISSUE TITLE
Val Loss 训练后面一直往上走

### ISSUE THREAD

**Issue by YangJing524** [bubbliiiing/deeplabv3-plus-pytorch#37]
博主大大，请教一下，这个是怎么造成的呢
![](https://img-blog.csdnimg.cn/667141d900da48f4bcd7c993c1ca034b.png)

**Comment by bubbliiiing** [comment]
过拟合了捏
